docs/migrate-matomo-4-to-5.md

```diff
@@ -0,0 +1,103 @@
+---
+category: Develop
+---
+# Migrate Plugin from Matomo 4.X to Matomo 5
+
+This migration guide covers how to do some migrations to make a plugin compatible with Matomo 5. A list of
+all changes in Matomo 5 can be found in the [Changelog](/changelog).
+
+## Create a new branch
+
+We recommend creating a new branch for your plugin that supports Matomo 5. For example `5.x-dev`. This way you will be able to make changes to your plugin for Matomo 4 and Matomo 5 and release independent versions for each of them. You can still publish updates to your plugin that supports Matomo 4 once you have published an update for a version that supports Matomo 5.
+
+## Adjust the required Matomo version
+
+For your plugin to be executed in Matomo 5 you first need to show it is compatible with Matomo 5 in your `plugin.json` file:
+
+* specify that your plugin requires Matomo 5 (the require for Matomo 4 used to be eg `"matomo": ">=4.0.0-b1,<5.0.0-b1"`). 
+* we also recommend increasing your plugin's major version number eg from `4.1.9` to `5.0.0`.
```

Comment:
Just a note here, no change needed for now — we use a mix of `e.g.` and `eg.` and `eg`. Since I haven't seen a language style guide for Matomo's docs, it won't matter much at the moment, just something to think about going forward. The correct way is `e.g.`, in US English they sometimes use it without the dots. One dot is incorrect.

Reply:
This part was copied from the old migration guide. Normally I use `e.g.`. But I agree that we should use something consistent...